index.html

```diff
@@ -52,114 +52,120 @@ <h1>Personal Profile</h1>
 				</div>
 				
 				<div class="sectionContent">
-					<p>I am a former registered nurse still licensed in both California and Texas. I am looking to make a career change into software development.</p>
+					<p>I am a former registered nurse looking to make a career change into software development.</p>
 				</div>
 			</article>
 			<div class="clear"></div>
 		</section>
         
         <section>
 			<div class="sectionTitle">
-				<h1>Education</h1>
+				<h1>Projects</h1>
 			</div>
 			
 			<div class="sectionContent">
                 <article>
-                    <h2>EDX Free Online Courses</h2>
-                    <p class="subDetails">July 2015 - July 2016</p>
-                    <p>Completed two Introduction to Computer Science courses.</p>
+                    <h2>BikeRacks</h2>
+                    <p class="subDetails">Interactive map</p>

```

Comment:
"Interactive Map" undersells this.

It's a platform for crowdsourcing bikerack location and quality